stable/xray/UPGRADE_NOTES.md

```diff
@@ -57,11 +65,11 @@ This file describes special upgrade notes needed at specific versions
           d. It will trigger the migration process
           Example:
           ```bash
-          $ helm install xray-new --set router.livenessProbe.enabled=false --set router.readinessProbe.enabled=false --set indexer.livenessProbe.enabled=false --set analysis.livenessProbe.enabled=false --set server.livenessProbe.enabled=false --set persist.livenessProbe.enabled=false --set indexer.readinessProbe.enabled=false --set analysis.readinessProbe.enabled=false --set server.readinessProbe.enabled=false --set persist.readinessProbe.enabled=false --set postgresql.enabled=false --set database.user=<OLD_PG_USERNAME> --set database.password=<OLD_PG_PASSWORD> --set database.url="postgres://<SERVICE_NAME_POSTGRES>:5432/xraydb?sslmode=disable" --set xray.mongoUsername=<OLD_MONGO_USERNAME> --set xray.mongoPassword=<OLD_MONGO_PASSWORD> --set xray.mongoUrl="mongodb://<SERVICE_NAME_MONGODB>:27017/?authSource=xray&authMechanism=SCRAM-SHA-1" --set xray.masterKey=<PREVIOUS_MASTER_KEY>  --set rabbitmq-ha.rabbitmqPassword=<PASSWORD> --set xray.jfrogUrl=<NEW_ARTIFACTORY_URL> --set  xray.joinKey=<JOIN_KEY>
+          $ helm install xray-new --set router.livenessProbe.enabled=false --set router.readinessProbe.enabled=false --set indexer.livenessProbe.enabled=false --set analysis.livenessProbe.enabled=false --set server.livenessProbe.enabled=false --set persist.livenessProbe.enabled=false --set indexer.readinessProbe.enabled=false --set analysis.readinessProbe.enabled=false --set server.readinessProbe.enabled=false --set persist.readinessProbe.enabled=false --set postgresql.enabled=false --set database.user=<OLD_PG_USERNAME> --set database.password=<OLD_PG_PASSWORD> --set database.url="postgres://<SERVICE_NAME_POSTGRES>:5432/xraydb?sslmode=disable" --set xray.mongoUsername=<OLD_MONGO_USERNAME> --set xray.mongoPassword=<OLD_MONGO_PASSWORD> --set xray.mongoUrl="mongodb://<SERVICE_NAME_MONGODB>:27017/?authSource=xray&authMechanism=SCRAM-SHA-1" --set xray.masterKey=<PREVIOUS_MASTER_KEY>  --set rabbitmq.auth.password=<PASSWORD> --set rabbitmq.enabled=true --set rabbitmq-ha.enabled=false --set xray.jfrogUrl=<NEW_ARTIFACTORY_URL> --set  xray.joinKey=<JOIN_KEY>
```

Comment:
We should move these blocks to be yaml file examples. It's hard to understand it.

Reply:
Changed.